src/services/DynamicPanelLoader.ts

```
@@ -0,0 +1,218 @@
+    // Load dependencies first
+    for (const depId of definition.dependencies) {
+      await this.loadPanel(depId);
+    }
+
+    // Dynamic import based on component name
+    try {
+      let component: React.ComponentType<any>;
+
+      switch (definition.component) {
+        case 'SessionStatsPanel':
+          const { default: SessionStatsPanel } = await import('../components/panels/SessionStatsPanel');
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)